### PR TITLE
Added 'start' goal

### PR DESCRIPTION
**Pull request by ajurasz**
Resolved issue #35, unfortunately there is very poor test coverage for this commit I'm just checking output for <code>deplo-start-all</code> goal.

### CODE REVIEW

File: maven-plugin/src/it/zed-deploy-start-all-it/verify.groovy [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import java.io.*
+import java.lang.*
+
+def found=false
```
Comment (hekonsek) [inline maven-plugin/src/it/zed-deploy-start-all-it/verify.groovy]: I propose to use Groovy power here :)

    def expectedOutputLine = new File(...).readLines().find{ it .contains("has been started with PID")}
    assert expectedOutputLine != null

File: maven-plugin/src/main/java/zed/maven/plugin/AbstractZedMojo.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import java.nio.file.Paths;
+import java.util.concurrent.Callable;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static java.lang.String.format;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static zed.utils.Mavens.artifactVersion;
+import static zed.utils.Mavens.localMavenRepository;
+
+public abstract class AbstractZedMojo extends AbstractMojo {
```
Comment (hekonsek) [inline maven-plugin/src/main/java/zed/maven/plugin/AbstractZedMojo.java]: +1 I wanted to asked to extract this to common base class.

**Comment by hekonsek** [comment]
Just fix the assertion to be more "Groovish" and we can merge it :) .

**Comment by ajurasz** [comment]
Not sure why travis build is failing

**Comment by hekonsek** [comment]
We generate too much output to STDOUT so Travis is killing the build. I will handle this.

**Comment by hekonsek** [comment]
Thanks for the contribution! I will cut the release soon :) .

**Pull request merged**